python_purify/tests.py

```diff
@@ -7,7 +7,7 @@
 bad = 'asdf'
 good = 'That\'s a nice hat.'
 
-jpurify = WordPurify(wordapi, live=False, debug=True)
+jpurify = WordPurify(wordapi, live=False, verbose=True)
```

Comment:
There is no `debug` flag.  Went with `verbose` instead:
https://github.com/kingthomasc/python-purify/blob/master/python_purify/core.py#L19

Before and after (api key changed to <api_key> in the output):

```
KevinsMacBookPro:python_purify kevin$ python tests.py 
Traceback (most recent call last):
  File "tests.py", line 10, in <module>
    jpurify = WordPurify(wordapi, live=False, debug=True)
TypeError: __init__() got an unexpected keyword argument 'debug'


KevinsMacBookPro:python_purify kevin$ python tests.py 
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.addtoblacklist&api_key=<api_key>&format=json&word=asdf&ds=1
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.getblacklist&api_key=<api_key>&format=json&ds=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.addtowhitelist&api_key=<api_key>&format=json&word=asdfasdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.getwhitelist&api_key=<api_key>&format=json
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.check&api_key=<api_key>&format=json&text=asdf&slink=0&semail=0&rsp=0&sphone=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.check&api_key=<api_key>&format=json&text=That%27s%20a%20nice%20hat.&slink=0&semail=0&rsp=0&sphone=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.checkcount&api_key=<api_key>&format=json&text=asdf%20That%27s%20a%20nice%20hat.%20asdf&slink=0&semail=0&rsp=0&sphone=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.checkcount&api_key=<api_key>&format=json&text=asdfasdf&slink=0&semail=0&rsp=0&sphone=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.return&api_key=<api_key>&format=json&text=asdf%20That%27s%20a%20nice%20hat.&slink=0&semail=0&rsp=0&sphone=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.replace&api_key=<api_key>&format=json&sphone=0&text=asdf%20That%27s%20a%20nice%20hat.&slink=0&semail=0&replacesymbol=8&rsp=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.removefromwhitelist&api_key=<api_key>&format=json&word=asdfasdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.checkcount&api_key=<api_key>&format=json&text=asdfasdf&slink=0&semail=0&rsp=0&sphone=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.removefromblacklist&api_key=<api_key>&format=json&word=asdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.check&api_key=<api_key>&format=json&text=asdf&slink=0&semail=0&rsp=0&sphone=0

KevinsMacBookPro:python_purify kevin$ python3 tests.py 
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.addtoblacklist&api_key=<api_key>&format=json&ds=1&word=asdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.getblacklist&api_key=<api_key>&format=json&ds=0
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.addtowhitelist&api_key=<api_key>&format=json&word=asdfasdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.getwhitelist&api_key=<api_key>&format=json
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.check&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&text=asdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.check&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&text=That%27s%20a%20nice%20hat.
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.checkcount&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&text=asdf%20That%27s%20a%20nice%20hat.%20asdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.checkcount&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&text=asdfasdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.return&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&text=asdf%20That%27s%20a%20nice%20hat.
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.replace&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&replacesymbol=8&text=asdf%20That%27s%20a%20nice%20hat.
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.removefromwhitelist&api_key=<api_key>&format=json&word=asdfasdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.checkcount&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&text=asdfasdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.removefromblacklist&api_key=<api_key>&format=json&word=asdf
http://api1.webpurify.com/services/rest/?method=webpurify.sandbox.check&api_key=<api_key>&format=json&semail=0&sphone=0&slink=0&rsp=0&text=asdf

```